py-polars/tests/unit/namespaces/test_datetime.py

```diff
@@ -14,162 +13,218 @@
     from polars.internals.type_aliases import TimeUnit
 
 
-def test_dt_strftime() -> None:
-    s = pl.Series("a", [10000, 20000, 30000], dtype=pl.Date)
-    assert s.dtype == pl.Date
-    expected = pl.Series("a", ["1997-05-19", "2024-10-04", "2052-02-20"])
-    assert_series_equal(s.dt.strftime("%F"), expected)
+@pytest.fixture()
+def date_2022_01_01() -> datetime:
+    return datetime(2022, 1, 1)
 
 
-def test_dt_year_month_week_day_ordinal_day() -> None:
-    s = pl.Series("a", [10000, 20000, 30000], dtype=pl.Date)
+@pytest.fixture()
+def date_2022_01_02() -> datetime:
+    return datetime(2022, 1, 2)
```

Comment:
This is a bit of fixture-overkill, in my eyes. Better to just specify these datetimes in-line rather than passing as a fixture.